website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,394 @@ To-do:
+
+### Example configuration
+
+To configure an Iceberg table materialization in dbt, refer to the example configuration:
+
+<File name='models/<modelname>.sql'>
+
+```sql
+
+{{
+  config(
+    materialized = "table",
+    table_format="iceberg",
+    external_volume="s3_iceberg_snow",
+    base_location_subpath="jaffle_shop_folder",
+  )
+}}
+
+select * from {{ ref('raw_orders') }}
+
+```
+
+</File>
+
+### Base location 
+
+If you were to create an Iceberg Table in the Snowflake query editor, you would be required to provide a `base_location` as an input. dbt abstracts away from that. The default behavior in dbt is to provide a `base_location` string that follows this convention:
+
+_dbt/{SCHEMA_NAME}/{MODEL_NAME}. 
+
+However, dbt allows users to configure a `base_location_subpath` that is empty by default but, when provided, will be concatenated to the end of the previously described pattern for `base_location` string generation.
+
+dbt does this to enforce best practices. With Snowflake-managed Iceberg format tables, the user owns and maintains the data storage of the tables in an external storage solution (the declared `external volume`). The base location declares where to write in the external volume. The Snowflake Iceberg catalog will keep track of your Iceberg table regardless of where the data lives within the external volume declared and what `base_location` is provided. However, passing anything into the `base_location` field, including an empty string, is possible.  If a project does not enforce an input and always allows empty strings, this could result in future technical debt because it will limit the ability to:
+
+- Navigate the underlying object store (S3/Azure blob)
+- Read Iceberg tables via an object-store integration
+- Grant schema-specific access to tables via object store
+- Use a crawler pointed at the tables within the external volume to build a new catalog with another tool
+
+In summary, dbt-snowflake does not support arbitrary definition of base_location for Iceberg tables. Instead, dbt, by default, writes your tables within a `_dbt/{SCHEMA_NAME}` prefix to ensure easier object-store observability and auditability.
+
+### Limitations
+
+There are some limitations to the implementation you need to be aware of:
+
+-  Using Iceberg tables with dbt, the result is that your query is materialized in Iceberg. However, often, dbt creates intermediary objects as temporary and transient tables for certain materializations, such as incremental ones. It is not possible to configure these temporary objects also to be Iceberg-formatted. You may see non-Iceberg tables created in the logs to support specific materializations, but they will be dropped after usage.
+- The `base_location_subpath` input is always appended to your schema name, and this behavior cannot be overridden. This ensures dbt can differentiate Iceberg model builds by environment.
```

Comment:
```suggestion
- The `base_location_subpath` input is always appended to your schema name, and this behavior cannot be overridden. This ensures dbt can differentiate Iceberg model builds by environment.
```

We should move this up to the base location paragraph as well as the bullet underneath about base_location_subpath field